booking_up_for_beauty/cbeer/booking_up_for_beauty.go

```diff
@@ -0,0 +1,46 @@
+package booking
+
+import (
+	"time"
+ 	"fmt"
+	)
+
+// Schedule returns a time.Time from a string containing a date
+func Schedule(date string) time.Time {
+	// Input example: 7/25/2019 13:45:00
+	layout := "1/2/2006 15:04:05"
+	t, _ := time.Parse(layout, date)
```

Comment:
You would want to handle the errors on bad inputs, however the tests do not require it. I wish there were error cases with bad inputs

Reply:
There are at least in other exercises.